molecule/default/molecule.yml

```diff
@@ -0,0 +1,33 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  enabled: False
+
+platforms:
+  - name: opengrok-${MOLECULE_DISTRO:-stretch}
+    image: debian:${MOLECULE_DISTRO:-stretch}
```

Comment:
I would prefer to use our java image based on Debian

Reply:
done